drivers/volume/portworx/portworx.go

```diff
@@ -32,8 +33,11 @@ const (
 	// namespace is the kubernetes namespace in which portworx daemon set runs
 	namespace = "kube-system"
 
-	//provisionerName is the name for the driver provisioner
+	// provisionerName is the name for the driver provisioner
 	provisionerName = "kubernetes.io/portworx-volume"
+
+	// pvcProvisionerAnnotation is the annotation on PVC which has the provisioner name
+	pvcProvisionerAnnotation = "volume.beta.kubernetes.io/storage-provisioner"
```

Comment:
this should be defined in k8s package, right? can we use it from there?

Reply:
Unfortunately, it's not exported from anywhere in kubernetes.

Reply:
ah okay. Just saw that we have defined it in sched-ops as well.